config/l3afd.cfg

```diff
@@ -46,25 +45,28 @@ ingress-entry-function-name: tc_ingress_root
 egress-entry-function-name: tc_egress_root
 
 [ebpf-chain-debug]
-addr: localhost:8899
-enabled: false
+addr: 0.0.0.0:8899
+enabled: true
 
 [l3af-configs]
-restapi-addr: localhost:53000
+restapi-addr: 0.0.0.0:7080
 
 [l3af-config-store]
 filename: /var/l3afd/l3af-config.json
 
 [mtls]
-enabled: true
+enabled: false
```

Comment:
Is this file used for other use-cases, or as an example config for users? If so we might want to consider these changes and make sure this is what we want users to use by default.